### ISSUE TITLE
getModuleDirectorySlow java.lang.Throwable: Assertion failed

### ISSUE THREAD

**Issue by sherif-tech** [wt-io-it/odoo-pycharm-plugin#97]
Plugin Version: 0.5.16
IDEA Version: PyCharm 2022.3

<details><summary>java.lang.Throwable: Assertion failed</summary>

```
java.lang.Throwable: Assertion failed
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:254)
	...
	at com.intellij.psi.search.FilenameIndex.getFilesByName(FilenameIndex.java:85)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModuleDirectorySlow$3(OdooModuleServiceImpl.java:136)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:860)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectorySlow(OdooModuleServiceImpl.java:134)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectory(OdooModuleServiceImpl.java:121)
	at at.wtioit.intellij.plugins.odoo.modules.index.OdooDeserializedModuleImpl.getDirectory(OdooDeserializedModuleImpl.java:64)
	at at.wtioit.intellij.plugins.odoo.WithinProject.call(WithinProject.java:15)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:38)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:42)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil$foreignResults$1.invoke(PyResolveImportUtil.kt:199)
	...
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
```
</details>

**Comment by ap-wtioit** [comment]
@sherif-tech it seems you are using an outdated plugin, the issue you reported here seems to be the same as #33 and was fixed in version 0.5.18, can you update the plugin?

**Comment by ap-wtioit** [comment]
Note: this was still happening in PyCharm 2022.3+ until it was fixed (again) in #67 with 0.6.10
